src/plugins/ui_core_views/cell_evaluation/evaluator.ts

```diff
@@ -104,13 +104,20 @@ export class Evaluator {
     this.formulaDependencies().addDependencies(positionId, dependencies);
   }
 
+  addDependencies(position: CellPosition, dependencies: Range[]) {
+    const positionId = this.encoder.encode(position);
+    this.formulaDependencies().addDependencies(positionId, dependencies);
+  }
+
   private updateCompilationParameters() {
     // rebuild the compilation parameters (with a clean cache)
     this.compilationParams = buildCompilationParameters(
       this.context,
       this.getters,
       this.computeAndSave.bind(this)
     );
+    this.compilationParams[2].updateDependencies = this.updateDependencies.bind(this);
```

Comment:
Yes, totally :) That was something I had in mind when we worked with Alexis on the evaluator but the it has been forgotten. It would greatly improve the readability of some part of the evaluation process

Reply:
https://www.odoo.com/web#cids=1&menu_id=4720&action=333&active_id=2328&model=project.task&view_type=form&id=3725622  👍

Reply:
I'm not sure it should be changed to an object, it's another discussion. However having right now a constant=2 that tells us what this [2] is would be more readable.